ParisBNF/abb/BNFabb20.xml

```diff
@@ -97,25 +97,27 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+             
+            </div>
             <div type="edition" xml:lang="gez">
                 <div type="textpart" n="1" subtype="folio">
                     <ab>
-                        <pb n="1r"/> ኆልቆሙ፡ ለእለ፡ ተጋብኡ። ለሕንፃ፡ ማኅፈድ፡ ፸ወ፪፡ መሳፍንት፡ እምደቂቀ፡ <persName ref="PRS0000">ሴም፡
-                        </persName> ፳ወ፭ብእሲ፡ ወእሉ፡ ኀደሩ፡ እም<placeName ref="LOC0000">አፍራጥስ፡ </placeName>ወእስከ፡
-                        <placeName ref="LOC0000">ሶርያ፡ </placeName>ወእስከ፡ ጽንፈ፡ ምሥራቅ፡ ወእምኔሆሙ፡ <placeName ref="LOC0000">ሶርያውያን፡
+                        <pb n="1r"/> ኆልቆሙ፡ ለእለ፡ ተጋብኡ። ለሕንፃ፡ ማኅፈድ፡ ፸ወ፪፡ መሳፍንት፡ እምደቂቀ፡ <persName ref="PRS8730Sem">ሴም፡
+                        </persName> ፳ወ፭ብእሲ፡ ወእሉ፡ ኀደሩ፡ እም<placeName ref="LOC3013Euphra">አፍራጥስ፡ </placeName>ወእስከ፡
+                        <placeName ref="LOC5833Syria">ሶርያ፡ </placeName>ወእስከ፡ ጽንፈ፡ ምሥራቅ፡ ወእምኔሆሙ፡ <placeName ref="ETH0000">ሶርያውያን፡
                         </placeName>ወ<placeName ref="LOC0000">አእፃዳተ፡ ዘረቢዐ፡ </placeName>ወደሴተ፡ <placeName ref="LOC0000">ወሐራምቃ፡ </placeName>
-                        <placeName ref="LOC0000">ከለዳውያን፡ </placeName>ወእሙንቱ፡ ሰብአ፡ <placeName ref="LOC0000">ባቢሎን፡
-                        </placeName>ወ<placeName ref="LOC0000">ፋርስ፡ </placeName> ወ<placeName ref="LOC0000">ኵራስን</placeName>፡
+                        <placeName ref="ETH0000">ከለዳውያን፡ </placeName>ወእሙንቱ፡ ሰብአ፡ <placeName ref="LOC1692Babylo">ባቢሎን፡
+                        </placeName>ወ<placeName ref="LOC3930Iran">ፋርስ፡ </placeName> ወ<placeName ref="LOC0000">ኵራስን</placeName>፡
                         ወ<placeName ref="LOC0000">ፌርያ</placeName>፡ ወ<placeName ref="LOC0000">ስንድ</placeName>፡
-                        ወ<placeName ref="LOC0000">ህንዳኬ</placeName>፡ ወ<placeName ref="LOC0000">የማን</placeName>፡
+                        ወ<placeName ref="LOC0000">ህንዳኬ</placeName>፡ ወ<placeName ref="LOC6387Yemen">የማን</placeName>፡
                         ወ<placeName ref="LOC0000">ጣይፍ</placeName>፡ ወ<placeName ref="LOC0000">ተማማ፡
-                        </placeName>ወ<placeName ref="LOC0000">፪አብሕርት</placeName>፡ ወዘመደ፡ <placeName ref="LOC0000">ዐረብ፡
-                        </placeName>ወእለ፡ የአምሩ፡ ጽሒፈ፡ ሆኅያት፡ ፳<placeName ref="LOC0000">እብራዊ፡
-                        </placeName>ወ<placeName ref="LOC0000">ሥየናዊ፡ </placeName>ወ<placeName ref="LOC0000">ፋርሳዊ</placeName>፡
-                        ወ<placeName ref="LOC0000">ካልዳዊ</placeName>፡ ወ<placeName ref="LOC0000">ሲኒ፡
-                        </placeName>ወ<placeName ref="LOC0000">ኃማዚ</placeName>፡ ወ<placeName ref="LOC0000">ዐረቢ፡ </placeName>
-                        ወሎሙ፡ እምአብሕርት፡ <placeName ref="LOC0000">ኤፍራጦስ</placeName>፡ ወፈለገ፡ <placeName ref="LOC0000">በለከ፡
+                        </placeName>ወ<placeName ref="LOC0000">፪አብሕርት</placeName>፡ ወዘመደ፡ <placeName ref="ETH1134Arab">ዐረብ፡
+                        </placeName>ወእለ፡ የአምሩ፡ ጽሒፈ፡ ሆኅያት፡ ፳<placeName ref="ETH1678Hebrew">እብራዊ፡
+                        </placeName>ወ<placeName ref="ETH0000">ሥየናዊ፡ </placeName>ወ<placeName ref="ETH0000">ፋርሳዊ</placeName>፡
```

Comment:
we do have LOC3930Iran for Fars and LOC5531Sana for San'a

Reply:
Fars in this text may refer to the ancient region of Persis, what is a smaller region in the central southern part of modern day Iran, but this is not clear. About Sanʿa I don't think, that we find the Yemeni capital in this text. In my dissertation I interpreted term śəyānāwi as Xian (=China) what is a misinterpretation of soryawi (=Syriac). Sanʿa is not possible, because we need to look for a distinct writing system.

Reply:
OK
Just insert the correction by Dorothea

Reply:
Actually I now opened the images at last (sorry late) and ms_i1 is of course not an `ms_i1 `at all, it is an additio (GuestText), added on protective quire by a very secondary hand. 
If your edition is that of the additio and not of a main text it is better to put it to the msDesc, inside of a `q` element within the addition description as we usually do with additions

Reply:
I created an additio a1 from ms_i1 as you said. I do not understand, what you mean with q element?

Reply:
(I do not yet see a commit with an additio - it should be in the physDesc, msContent has then only 2 texts)

Reply:
If I understand you right, I shall insert the quoted text into the q element within the addition item. I created two q elements now for the text and for the translation. I will commit in a minute. I hope, you will receive it.

Reply:
Make sure you have read https://betamasaheft.eu/Guidelines/?id=additionsVaria
`If you are providing a translation of the added text, you can use the same method, but add @type with value translation to the translation <q>[↗](https://betamasaheft.eu/Guidelines/?id=q).`

Reply:
Yes, I did it that way. I created an additional q element for the translation following the instruction in the guidelines.